package.json

```diff
@@ -21,20 +24,59 @@
   },
   "homepage": "https://github.com/cachilders/backpat#readme",
   "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-plugin-istanbul": "^3.0.0",
+    "babel-preset-es2015": "^6.18.0",
+    "babel-register": "^6.18.0",
     "chai": "^3.5.0",
     "chai-spies": "^0.7.1",
     "coveralls": "^2.11.15",
+    "cross-env": "^3.1.3",
     "eslint": "^3.8.1",
     "eslint-plugin-flowtype": "^2.25.0",
+    "ghooks": "^1.3.2",
     "mocha": "^3.1.2",
     "npm-run-all": "^3.1.2",
-    "nyc": "^10.0.0"
+    "nyc": "^10.0.0",
+    "rimraf": "^2.5.4"
   },
   "nyc": {
     "check-coverage": true,
     "functions": 90,
     "lines": 90,
-    "statements": 90
+    "statements": 90,
```

Comment:
These check-coverage limits are thresholds where if any of the coverage percentages fall below these values, test (nyc specifically) will exit with a value of 2. I recommend we lower these because it's going to be difficult to commit otherwise. See below for explanation of why.